CMakeLists.txt

```diff
@@ -329,7 +329,7 @@ set(KTX_BUILD_DIR "${CMAKE_BINARY_DIR}")
 
 if(WIN32)
     target_link_libraries(
-        ktx
+        ktx PRIVATE
```

Comment:
This and related lines will have to change after PR #324 is merged as, if the built library is a static library(an option added by PR #324) libzstd will be required by the application.

Reply:
Wasn't aware of this. I'd need to test.

Reply:
libktx has references to functions in libzstd. If you are building a shared library these will cause libzstd to be linked into the shared libktx. If static, the app needs to link with libzstd to satisfy the references.